govcd/sample_govcd_test_config.yaml

```diff
@@ -102,6 +102,8 @@ vcd:
     storageProfile:
         # First storage profile (mandatory)
         storageProfile1: Development
+        # ID of storage profile. If omitted, some tests will be skipped.
+        storageProfile1ID: 8263c1b5-6a40-404f-84a3-72343fce7c20
```

Comment:
Thanks for adding tests!
I will just ask to try and avoid adding this value to config. You could in fact retrieve a storage profile ID in your test  by using some of existing functions below to get ID by using Name:
```
func (vdc *Vdc) FindStorageProfileReference(name string) (types.Reference, error) {
func (adminOrg *AdminOrg) GetAllStorageProfileReferences(refresh bool) ([]*types.Reference, error) {
```

That way we don't have to introduce a new variable into test config, which is a struggle to populate.

Reply:
👌

Reply:
fixed